packages/components/src/color-palette/index.tsx

```diff
@@ -284,22 +342,45 @@ export default function ColorPalette( {
 					) }
 				/>
 			) }
-			<Component
-				clearable={ clearable }
-				clearColor={ clearColor }
-				colors={ colors }
-				onChange={ onChange }
-				value={ value }
-				actions={
-					!! clearable && (
-						<CircularOptionPicker.ButtonAction
-							onClick={ clearColor }
-						>
-							{ __( 'Clear' ) }
-						</CircularOptionPicker.ButtonAction>
-					)
-				}
-			/>
+			{ __experimentalHasMultipleOrigins ? (
+				<MultiplePalettes
+					{ ...paletteCommonProps }
+					colors={ colors as PaletteObject[] }
+				/>
+			) : (
+				<SinglePalette
+					{ ...paletteCommonProps }
+					colors={ colors as ColorObject[] }
+				/>
+			) }
 		</VStack>
 	);
 }
+
+/**
+ * Allows selecting from given colors.
```

Comment:
Maybe this JSDoc block isn't what you [had in mind](https://github.com/WordPress/gutenberg/pull/44632#discussion_r999310616). 

I didn't see a description in [README.md](https://github.com/WordPress/gutenberg/blob/5a82cf5723b0feb0de9ec23e267f6b190d2a5ad0/packages/components/src/color-palette/README.md), so I added this, which might be completely off 😄

Reply:
It looks good for now, we can always refine it before merging — thank you!

Reply:
Thanks, Marco!